src/AltinnCore/Common/Enums/FileEditorMode.cs

```diff
@@ -43,5 +43,20 @@ public enum FileEditorMode
         /// All the files in the Test folder
         /// </summary>
         Test = 6,
+
+        /// <summary>
+        /// All the files in the Model folder
+        /// </summary>
+        Calculation = 7,
+
+        /// <summary>
+        /// All the files in the Resource folder
+        /// </summary>
+        Dynamics = 8,
+
+        /// <summary>
+        /// All the files in the Test folder
```

Comment:
This summary is the same as the one for test, and should be updated

Reply:
Updated the duplicate comments in this file